CTP/sql/bin/run_memory.sh

```diff
@@ -0,0 +1,202 @@
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+#
+#Copyright (c) 2016, Search Solution Corporation? All rights reserved.
+#
+#Redistribution and use in source and binary forms, with or without 
+#modification, are permitted provided that the following conditions are met:
+#
+#  * Redistributions of source code must retain the above copyright notice, 
+#    this list of conditions and the following disclaimer.
+#
+#  * Redistributions in binary form must reproduce the above copyright 
+#    notice, this list of conditions and the following disclaimer in 
+#    the documentation and/or other materials provided with the distribution.
+#
+#  * Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products 
+#    derived from this software without specific prior written permission.
+#
+#THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+#INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+#DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+#SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
+#SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, 
+#WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+#USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+export CTP_HOME=$(cd $(dirname $(readlink -f $0))/../..; pwd)
+scenario_category=""
+config_file_main=""
+memory_log="memory.log"
+
+alias ini="sh ${CTP_HOME}/bin/ini.sh"
+
+
+function usage ()
+{
+     echo ""
+     echo "Usage: $0 -s <type of scenario>
+           -f <test configuration file"
+     echo ""
+}
+
+while getopts "s:f:h:" opt; do
+     case $opt in
+     s)
+         scenario_category="$OPTARG"
+         ;;
+     f)
+         config_file_main="$OPTARG"
+         ;;
+     \?)
+         usage
+         exit 1
+         ;;
+     esac
+done
+
+function clean_results()
+{
+   curDir=`pwd`
+   if [ ! -d "$CTP_HOME/result" ];then
+       cd $CTP_HOME
+       mkdir -p result
+   else
+       cd $CTP_HOME/result
+       if [ -d "memory" ];then
+          rm ./memory/*
+       fi
+   fi
+
+   cd $curDir
+}
+
+
+function rename_process()
+{
+   curDir=`pwd`
+
+   if [ ! -d "$CUBRID/bin" ];then
+      echo "Please confirm your build is installed correctly!"
+      exit 1
+   fi
+
+   cd $CUBRID/bin
+   mv cub_server server.exe
```

Comment:
as is: "mv cub_server server.exe"
to be: If server.exe exists, do not move again

Note: cas.exe is similar.